backend/src/Edna/Upload/Web/Types.hs

```diff
@@ -14,19 +14,19 @@ module Edna.Upload.Web.Types
 
 import Universum
 
-import qualified Data.Swagger as S
+import qualified Data.HashMap.Strict.InsOrd as Map
```

Comment:
Usually `Map` is used for `Data.Map` or `Data.Map.Strict`. If I see `Map.*` in code, I usually assume that it's from `containers`. So I would expand this name, maybe `InsMap` for example.